feature-utils/poly-analysis/src/report/ministories-status-report.js

```diff
@@ -17,10 +17,8 @@ export default class MinistoriesStatusReport {
             };
         });
         this.active = this._ministoriesData.length > 0;
-    }
-
-    get title() {
-        return "Mini-stories status";
+        this.title = this._title;
```

Comment:
This could be done slightly differently. Since you are assigning the title in the reportWrapper, where you make a `new MinistoryStatusReport(ministories)` it would be a good idea to add the title and description in the constructor of the class. So the class constructor would look something like:
```
constructor({ministories, title, description}) {
        this._ministoriesData = ministories.map((ministory) => {
            return {
                id: ministory.constructor.name,
                status: ministory.active ? "ACTIVE" : "INACTIVE",
            };
        });
        this.active = this._ministoriesData.length > 0;
        this.title =title;
        this.description = description
    }
```
and the class instantiation in reportWrapper would be:
```
const statusReport = new MinistoriesStatusReport({ministories:[
        ...computedReportStoriesList,
        ...computedMinistories,
    ], title: i18n.t("report:mini.story.status"),
description: i18n.t("report:mini.story.status.details")});
```
and you won't need the next two lines where you assign the title and description to the status report :)